RC4: NPE in FingerprintActivity.onStop (FingerprintActivity:169)

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performStopActivityInner (ActivityThread.java:3903)
  at android.app.ActivityThread.handleStopActivity (ActivityThread.java:3952)
  at android.app.ActivityThread.-wrap25 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1530)
  at android.os.Handler.dispatchMessage (Handler.java:102)
  at android.os.Looper.loop (Looper.java:154)
  at android.app.ActivityThread.main (ActivityThread.java:6186)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:889)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:779)
Caused by: java.lang.NullPointerException: 
  at com.owncloud.android.ui.activity.FingerprintActivity.onStop (FingerprintActivity.java:169)
  at android.app.Instrumentation.callActivityOnStop (Instrumentation.java:1290)
  at android.app.Activity.performStop (Activity.java:6859)
  at android.app.ActivityThread.performStopActivityInner (ActivityThread.java:3900)
```

--- a/src/main/java/com/owncloud/android/ui/activity/FingerprintActivity.java
+++ b/src/main/java/com/owncloud/android/ui/activity/FingerprintActivity.java
@@ -166,7 +166,9 @@ public class FingerprintActivity extends AppCompatActivity {
     @Override
     public void onStop(){
         super.onStop();
-        cancellationSignal.cancel();
+        if(cancellationSignal != null) {
+            cancellationSignal.cancel();
+        }
     }
 
     /**

